src/main/kotlin/org/neo4j/graphql/GraphQLExtensions.kt

```diff
@@ -60,11 +64,8 @@ fun GraphQLFieldsContainer.relationshipFor(name: String): RelationshipInfo? {
                 ?: throw IllegalStateException("Field $field needs an @relation directive")
     }
 
-    // TODO direction is depending on source/target type
-
-    val relInfo = relDetails(fieldObjectType) { argName, defaultValue -> relDirective.getArgument(argName, defaultValue) }
+    val relInfo = relDetails(fieldObjectType, relDirective)
 
-    val inverse = isRelFromType && fieldObjectType.getFieldDefinition(relInfo.startField)?.name != this.name
```

Comment:
where is this bit handled now?

Reply:
this was the fix for issue #45

Reply:
I moved it to https://github.com/Andy2003/neo4j-graphql-java/blob/bugfix/%2398-fix-relationship-mapping/src/main/kotlin/org/neo4j/graphql/GraphQLExtensions.kt#L57 since it is only relevant for rich relations